app.js

```diff
@@ -24,12 +27,22 @@ mongoose.connect(`mongodb+srv://${MONGO_USERNAME}:${MONGO_PASSWORD}@cluster0-70y
 	console.log("ERROR", err.message);
 });
 
-
 app.use(bodyParser.urlencoded({extended:true})); 
 app.set("view engine", "ejs");	
 
+// Passport Configuration
+app.use(require("express-session")({
+	secret: "Chingu is cool",
```

Comment:
Should we get the "secret" string from an environment variable?

Reply:
Yeah im still unsure on this. Feel free to change if you think it is best practise.